custom_tests/srcs/ubpf_test_custom_local_function_stack_size.cc

```diff
@@ -5,8 +5,8 @@
 #include <iostream>
 #include <memory>
 #include <stdint.h>
-#include <vector>
 #include <string>
+#include <string.h>
```

Comment:
curious as to why both <string> and <string.h>

Reply:
Great question! We need that for `memset`. I just pushed a revision that uses `cstring` instead. Does that seem more reasonable?